### PR TITLE
Example Liquid Swipe

### PR DESCRIPTION
**Pull request by Szymon20000**

### CODE REVIEW

File: Common/cpp/NativeModules/NativeReanimatedModule.cpp [hunk 1]
```diff
@@ -323,26 +329,38 @@ void NativeReanimatedModule::unregisterMapper(jsi::Runtime &rt, int id) {
...
+    this->workletModule = std::shared_ptr<BaseWorkletModule>(new WorkletModule(
+      sharedValueRegistry,
+      applierRegistry,
+      workletRegistry,
+      eventPtr,
+      this->errorHandler));
+  } else {
+      std::dynamic_pointer_cast<WorkletModule>(this->workletModule)->setEvent(eventPtr);
+  }
+  SpeedChecker::checkSpeed("Event:", [=] () {
```
Comment (karol-bisztyga) [inline Common/cpp/NativeModules/NativeReanimatedModule.cpp:361]: speed check in general purpose code

**Comment by terrysahaidak** [comment]
It seems like I cannot build it on Android, some problems with linking yoga (again). Is this related to the PR or it's just me?

**Comment by Szymon20000** [comment]
It should work now. However, you need to call "git checkout -- Example/node_modules" after yarn.

**Comment by terrysahaidak** [comment]
Thanks. Yep, it's working now. So, RN currently doesn't provide a way to link TurboModule enabled libraries (at least its NDK part)?

**Pull request merged**